packages/next-templates/src/app/globals.css

```diff
@@ -343,7 +373,7 @@ utrecht-alert
 }
 
 .example-toptask-nav--grid {
-  padding-inline: var(--voorbeeld-space-block-cat);
+  padding-inline-end: var(--voorbeeld-space-inline-cat);
```

Comment:
Ik weet het niet zeker, maar zou dit niet een `inline-gap` kunnen zijn in de `.voorbeeld-column-layout`? Die dient dan als minimumafstand, je kunt alsnog met alignemnt de openingstijden verder opzij zetten.